src/pages/MediaHolder.php

```diff
@@ -2,18 +2,20 @@
 
 namespace nglasl\mediawesome;
 
+use Page;
 use SilverStripe\Forms\DropdownField;
 use SilverStripe\Forms\GridField\GridField;
 use SilverStripe\Forms\GridField\GridFieldConfig_RecordEditor;
 use SilverStripe\Forms\GridField\GridFieldDeleteAction;
 use SilverStripe\Forms\ReadonlyField;
+use SilverStripe\ORM\DataList;
 
 /**
  *	Displays media holder/page children, with optional date/tag filters.
  *	@author Nathan Glasl <nathan@symbiote.com.au>
  */
 
-class MediaHolder extends \Page {
```

Comment:
Cheers for the PR. Is there a specific reason for this change? I used to have it like this, but remember changing it for a specific reason (`Page` is only ever referenced from up here too). I assume it's causing issues with the static analysis?

Reply:
No practical reason for this change!

I have noticed that code on my current project that uses `\Exception` or `\Page` is more error-prone with maintenance though. Someone always comes along and does `Exception` or `Page` without the backslash and the program crashes.

Another benefit to using `use` over fully qualified names is that when namespaces / classes change, you will only be modifying all the `use` statements at the top rather than various lines of code throughout the program, so it's easier to reason about what actually changed.

All this being said, if you'd prefer to keep the \Page style. I'll revert this. (or just leave this PR as-is if you plan to close it and make sweeping phpDoc changes)

Reply:
No worries! I remember changing it primarily due to an issue I encountered at the time, but I'll take a look at this again now that the SS4 upgrade is complete :)